pkg/persistentvolumeclaim/debug_cstor_test.go

```diff
@@ -66,3 +68,53 @@ func TestDebugCstorVolumeClaim(t *testing.T) {
 		})
 	}
 }
+
+func Test_displayPVCEvents(t *testing.T) {
+	k, _ := client.NewK8sClient("openebs")
```

Comment:
Create a fake-client for UT for each test-suite.

Reply:
UTs are currently not written for this package, I will raise a separate PR with all units at the end of this feature. They are just added to pass the golangci for now.